Sources/ContainerizationArchive/CArchive/include/archive.h

```diff
@@ -613,7 +613,7 @@ __LA_DECL int archive_read_data_block(struct archive *a,
  * Some convenience functions that are built on archive_read_data:
  *  'skip': skips entire entry
  *  'into_buffer': writes data into memory buffer that you provide
- *  'into_fd': writes data to specified filedes
+ *  'into_fd': writes data to specified file descriptor
```

Comment:
This is a vendored header from another project so I'd prefer we don't modify it.